x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/related_integrations/related_integrations.cy.ts

```diff
@@ -190,12 +188,10 @@ describe('Related integrations', { tags: ['@ess', '@serverless', '@brokenInServe
       });
     });
 
-    // TODO: https://github.com/elastic/kibana/issues/161540
-    // Flaky in serverless tests
-    // @brokenInServerless tag is not working so a skip was needed
-    describe.skip('rule details', { tags: ['@brokenInServerless'] }, () => {
+    describe('rule details', () => {
```

Comment:
Now when it's unskipped it ESS, let's run the spec in the flaky runner and include the link in the PR.

Reply:
Added a link to the description. 100 runs were successful.